src/Core/src/Platform/Windows/SearchBarExtensions.cs

```diff
@@ -81,14 +93,22 @@ public static void UpdateMaxLength(this AutoSuggestBox platformControl, ISearchB
 				platformControl.Text = currentControlText.Substring(0, searchBar.MaxLength);
 		}
 		
-		public static void UpdateIsReadOnly(this AutoSuggestBox nativeControl, ISearchBar searchBar, MauiSearchTextBox? queryTextBox)
+		public static void UpdateIsReadOnly(this AutoSuggestBox platformControl, ISearchBar searchBar, MauiSearchTextBox? queryTextBox)
 		{
+			if (queryTextBox == null)
+				queryTextBox = platformControl.GetFirstDescendant<MauiSearchTextBox>();
+
 			if (queryTextBox == null)
 				return;
+
+			queryTextBox.IsReadOnly = searchBar.IsReadOnly;
```

Comment:
This looks like a  previous wrong merge since nothing was done here. Applied changes to fix applying the IsReadOnly property.